lms/envs/common.py

```diff
@@ -345,9 +345,12 @@
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
 
-    # Show Language selector.
+    # Show language selector in header.
     'SHOW_LANGUAGE_SELECTOR': False,
 
+    # Show language selector in footer.
+    'SHOW_FOOTER_LANGUAGE_SELECTOR': False,
```

Comment:
I think it would be unexpected if the meaning of the `SHOW_LANGUAGE_SELECTOR: True` changed from 'show the language selector in the header' to 'show the language selector in both the header and footer'. Would it be acceptable instead to create a new setting called `LANGUAGE_SELECTOR`, which could take an enum like you have described, and deprecate the old `SHOW_LANGUAGE_SELECTOR` setting? Do we have a deprecation policy for changes like these?